pkg/apis/core/types.go

```diff
@@ -2189,37 +2189,31 @@ type Container struct {
 }
 
 // ProbeHandler defines a specific action that should be taken in a probe.
-// This type has a strong relationship to LifecycleHandler - overlapping fields
```

Comment:
there is no need to keep this information. Backward compatibility requirements will take care of this

Reply:
This was forward-looking - if we use the same name it should probably be the same schema?  Maybe not,  but I assumed so...

Reply:
hm, this is guaranteed by the matching types like `ExecAction`.  The only thing this comment help with is the presence of fields, which is also guaranteed by back-compat requirement.

Reply:
I'm going to resolve it.  I think I am wrong